gradle/dependencies.gradle

```diff
@@ -158,7 +158,7 @@ versions += [
   // https://github.com/scalameta/scalafmt/releases/tag/v3.1.0.
   scalafmt: "3.7.14",
   scalaJava8Compat : "1.0.2",
-  scoverage: "1.9.3",
+  scoverage: "1.4.11",
```

Comment:
Why did you need to downgrade this version?

Reply:
this is the last 1.x version with 2.13 scala
https://repo1.maven.org/maven2/org/scoverage/scalac-scoverage-plugin_2.13.8/

there's one more for 2.13 scala: 2.13.9, but it only has 2.x versions under that
https://repo1.maven.org/maven2/org/scoverage/scalac-scoverage-plugin_2.13.9/

Reply:
I'm trying to upgrade to 2.x scoverage (with plugin upgrade too)